src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,104 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type AddMovie = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie: React.FC<AddMovie> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [form, setForm] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  } as Movie);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    setCount(count + 1);
+    setForm({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+
+    onAdd(form);
```

Comment:
first, you send the form, after - do reset
```suggestion
    onAdd(form);
    
    setForm({
      title: '',
      description: '',
      imgUrl: '',
      imdbUrl: '',
      imdbId: '',
    });
```